### PR TITLE
Added `dwarfFortress` spinner

### PR DESCRIPTION
**Pull request by dogversioning**
This adds an extremely silly spinner for when you've progressed too far and unleashed something extremely Fun. Based off of Dwarf Fortress's ASCII tileset.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,6 +40,7 @@
 		"json"
 	],
 	"devDependencies": {
+		"@types/node": "^17.0.41",
```
Comment (dogversioning) [inline package.json:43]: Re: this change, it looks like there was a breaking change to this package - see [this stackoverflow post](https://stackoverflow.com/questions/75386478/error-in-node-modules-types-node-ts4-8-util-d-ts148542-error-ts1005-ex). Happy to adopt another approach if you want.

Also - happy to uprev the min version to node 8 if you're so inclined for deprecation/unit test reasons, but didn't want to do it unilaterally.

File: spinners.json [hunk 2]
```diff
@@ -1464,5 +1464,143 @@
 			"▰▰▰▰▰▰▰",
 			"▰▱▱▱▱▱▱"
 		]
+	},
+	"dwarfFortress": {
+		"interval": 100,
```
Comment (sindresorhus) [inline spinners.json:1469]: The animation feels a bit slow. 80 seems a bit smoother.

**Comment by sindresorhus** [comment]
This is a fun one 👍

File: spinners.json (hunk 2 above)
Comment (dogversioning) [reply spinners.json]: yep, makes sense.

**Pull request merged**